test/Freezr.spec.js

```diff
@@ -0,0 +1,30 @@
+import {
+  buildMockObjectFrozen,
+  buildMockObjectDeepFrozen,
+  buildMockArrayFrozen,
+  // buildMockArrayDeepFrozen,
+  expectToBeImmutableAndThrowError
+} from './helpers'
+
+describe('freezr', function () {
+  it('returns an immutable object', () => {
+    const { frozen } = buildMockObjectFrozen()
+    expectToBeImmutableAndThrowError(frozen)
+  })
+
+  it('returns an immutable array', () => {
+    const { frozen } = buildMockArrayFrozen()
+    expectToBeImmutableAndThrowError(frozen)
+  })
+
+  describe('deepFreezr', function () {
```

Comment:
test name: deepFreeze.
it shouldn't be nested as it's being exported in freezer.js